src/transform/__testfixtures__/CSSinJS-broken-pattern.output.js

```diff
@@ -0,0 +1,5 @@
+const PriceElement = styled('div')(
+  ({ theme }) => css`
+    border-radius:${3}px`/*TOGGLE_REMOVE*/};
```

Comment:
Are we really expecting this `/*TOGGLE_REMOVE*/` to be there after the cleanup?

Reply:
Yes, this example highlights the "broken" pattern, as a reminder for the patterns that don't work yet.